solc/CommandLineInterface.cpp

```diff
@@ -1200,9 +1201,14 @@ bool CommandLineInterface::processInput()
 		};
 		if (countEnabledOptions(nonAssemblyModeOptions) >= 1)
 		{
+			auto optionEnabled = [&](string const& name){ return m_args.count(name) > 0; };
+			auto enabledOptions = boost::copy_range<vector<string>>(nonAssemblyModeOptions | boost::adaptors::filtered(optionEnabled));
+
 			serr() << "The following options are invalid in assembly mode: ";
-			serr() << joinOptionNames(nonAssemblyModeOptions) << ". ";
-			serr() << "Optimization is disabled by default and can be enabled with --" << g_argOptimize << "." << endl;
+			serr() << joinOptionNames(enabledOptions) << ". ";
```

Comment:
```suggestion
			serr() << joinOptionNames(enabledOptions) << ".";
```

Reply:
Yeah, this space should be moved to the next line. The code style check just noticed that it generates trailing whitespace in an error message in one of the tests.

Fixed.